specification/dartLangSpec.tex

```diff
@@ -18083,7 +18083,8 @@ \subsection{Yield-Each}
   if the class of $o$ is not a subtype of \code{Iterable<$T_f$>}.
   Otherwise
 \item
-  The method \code{iterator} is invoked upon $o$ returning an object $i$.
+  The getter \code{iterator} is invoked upon $o$ returning an object $i$.
+  Otherwise
```

Comment:
I should follow the norm here.
I was probably trying to avoid introducing new syntax (which I think is a bad choice in general, semantics should be explained at the semantic level without dipping back into syntax). But that's a much larger rewrite to avoid that.